ibis/config_init.py

```diff
@@ -50,3 +50,7 @@
 
 with cf.config_prefix('clickhouse'):
     cf.register_option('temp_db', '__ibis_tmp', clickhouse_temp_db_doc)
+
+
+with cf.config_prefix('bigquery'):
+    cf.register_option('partition_col', 'PARTITIONTIME')
```

Comment:
Should there also be one for `NATIVE_PARTITION_COL`?

Reply:
that's not a thing a user can reasonably set, so why would it be a settable option?

Reply:
I'm thinking of a scenario where BigQuery changes the name and users don't want to rely on ibis devs to update the repo to deal with it.

Reply:
In that case the user can clobber the module level variable `NATIVE_PARTITION_COL` in that case.

To have the value set in config_init suggests its configurable, which it is not.